pkg/grpc/api.go

```diff
@@ -1715,6 +1723,135 @@ func (a *api) SetMetadata(ctx context.Context, in *runtimev1pb.SetMetadataReques
 	return &emptypb.Empty{}, nil
 }
 
+func (a *api) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	if in.WorkflowReference.InstanceID == "" {
+		err := status.Errorf(codes.InvalidArgument, fmt.Sprintf(messages.ErrMissingOrEmptyInstance))
```

Comment:
Why fmt.Sprintf here if there's no argument passed? Can you just pass `messages.ErrMissingOrEmptyInstance`?